addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -366,13 +368,13 @@ def _import_retrieve_and_fill_partner_bank_details(self, invoice, bank_details):
         banks_to_create = []
         acc_number_partner_bank_dict = {
             bank.sanitized_acc_number: bank
-            for bank in self.env['res.partner.bank'].search(
+            for bank in Bank.search(
```

Comment:
This is a useless diff that will only cause potential conflicts.
It also makes it less readable IMO. `Bank` could be `res.partner.bank` or `res.bank` for instance.

Reply:
`ResPartnerBank` will be more precise. The code is the same in master and if there are conflicts I'll solve them.